x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/action/ShardFollowNodeTaskTests.java

```diff
@@ -228,6 +229,35 @@ public void testReceiveRetryableError() {
         assertThat(status.leaderGlobalCheckpoint(), equalTo(63L));
     }
 
+    public void testEmptyShardChangesResponseShouldClearFetchException() {
```

Comment:
I think we generilize this and test that any response (empty or not) clears errors.

Reply:
This is tested in `testReceiveRetryableError()`.